tablet-report-parser/README.md

```diff
@@ -103,3 +108,13 @@ SELECT tablename,uniq_tablet_count as uniq_tablets,
         WHERE sst_table_mb > 5000
         ORDER by sst_table_mb desc;
 ```
+
+### "Advanced" Run options:
+By default, the script will automatically pipe the  output stream to sqlite3, and create a sqlite database named <input-file>.sqlite.
```

Comment:
Nit: Duplicate space between "the" and "output".

Reply:
Fixed and merged into main.